src/frontend/src/handler/mod.rs

```diff
@@ -131,6 +132,8 @@ pub(super) async fn handle(session: Arc<SessionImpl>, stmt: Statement) -> Result
             }
             create_index::handle_create_index(context, name, table_name, columns).await
         }
+        Statement::StartTransaction { .. } => Ok(PgResponse::empty_result(START_TRANSITION)),
```

Comment:
Pls add a Todo comment here to note it's intended and not a final implementation

Reply:
All Done